Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.17
```

Comment:
This needs to be: registry.ci.openshift.org/openshift/release:golang-1.17

Reply:
Updated